lib/c-api/src/wasm_c_api/engine.rs

```diff
@@ -23,7 +23,6 @@ use wasmer_engine_universal::Universal;
 ///
 /// This is a Wasmer-specific type with Wasmer-specific functions for
 /// manipulating it.
-#[cfg(feature = "compiler")]
```

Comment:
Undelete this :)

Reply:
Which is used by this method:
```
pub extern "C" fn wasmer_is_compiler_available(compiler: wasmer_compiler_t) -> bool {
    match compiler {
        wasmer_compiler_t::CRANELIFT if cfg!(feature = "cranelift") => true,
        wasmer_compiler_t::LLVM if cfg!(feature = "llvm") => true,
        wasmer_compiler_t::SINGLEPASS if cfg!(feature = "singlepass") => true,
        _ => false,
    }
}
```

Reply:
Please, keep `#[cfg(feature = "compiler")]` here. But add:

```
use super::super::engine:{wasm_config_t, wasmer_engine_t};
#[cfg(feature = "compiler")]
use super::super::engine::wasmer_compiler_t;
```

in `lib/c-api/src/wasm_c_api/unstable/engine.rs`. It's a bug here.

Thank you very much!

Reply:
I added those compiler conditions, I had to add some others to fix more build errors though?